.eslintrc.js

```diff
@@ -134,6 +134,8 @@ module.exports = {
       extends: [
         'plugin:@typescript-eslint/eslint-recommended',
         'plugin:@typescript-eslint/recommended',
+        // 'plugin:@typescript-eslint/strict', TODO: See what errors come here
```

Comment:
They recommend either using `plugin:@typescript-eslint/recommended` or `plugin:@typescript-eslint/strict`, will post the diff in another branch in a bit.